apps/hash-graph/src/subcommand/server.rs

```diff
@@ -67,10 +105,18 @@ pub struct ServerArgs {
     #[clap(flatten)]
     pub pool_config: DatabasePoolConfig,
 
-    /// The address the REST client is listening at.
+    /// The address the REST server is listening at.
     #[clap(flatten)]
     pub api_address: ApiAddress,
 
+    /// Enable the experimental RPC server.
+    #[clap(long, default_value_t = false, env = "HASH_GRAPH_RPC_ENABLED")]
+    pub rpc_enabled: bool,
+
```

Comment:
Alternatively we could disable it if no address is set

Reply:
I thought about that as well, but I wanted to give the opportunity to have a default for the port and address as well (also I was unsure how clap would handle a `clap(flatten)` on an `Option<RpcAddress>` - not that I tried). I feel like this way you can still have a default which is nice.

I am not opposed to make it optional tho.

Reply:
I hear your argument about defaults. This also again implies the issue of unsetting the address if it's set in `.env`. I'm fine with leaving it. At some point we need to overhaul the environment variables anyway.